viewer/viewer/templates/viewer/search_side_bar.html

```diff
@@ -1,4 +1,4 @@
-
+<!-- 
```

Comment:
Better to delete this than comment out the whole file.

Reply:
I plan to in the near future, however there is a small functionality I am still referencing. It should be removed by the next major PR